include/libdnf/conf/config.hpp

```diff
@@ -20,15 +20,49 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #ifndef LIBDNF_CONF_CONFIG_HPP
 #define LIBDNF_CONF_CONFIG_HPP
 
+#include "config_parser.hpp"
 #include "option_binds.hpp"
+#include "option.hpp"
+#include "vars.hpp"
+#include "../logger/logger.hpp"
+
+#include <fmt/format.h>
 
 namespace libdnf {
 
 /// Base class for configurations objects
+template <Option::Priority default_priority>
 class Config {
 public:
     OptionBinds & opt_binds() noexcept { return binds; }
 
+    void load_from_parser(
```

Comment:
1. Could you move the implementation of the method to the .cpp file?
    And move `#include <fmt/format.h>` to .cpp file. (btw It solves problem with compiling swig ruby bindings on my PC)

Reply:
Done.